### ISSUE TITLE
Image not working as expected

### ISSUE THREAD

**Issue by PeterKottas** [Niels-IO/next-image-export-optimizer#102]
~~This might be connected to https://github.com/Niels-IO/next-image-export-optimizer/issues/45 but I am not sure.~~

I noticed that when I load an image ~~with priority~~, it won't load optimized images anymore. Here's a picture without priority:
![images2](https://user-images.githubusercontent.com/10601911/219515212-5543eccf-d8a7-47c0-b8b1-0e828fa0f1bf.jpg)
Here's one with:
![images](https://user-images.githubusercontent.com/10601911/219515191-3e2dde65-e2d9-4233-a882-d37ebe483e54.jpg)
Notice there's no webp and all images are the same. I wonder if the way I load it might be causing it. This is my setup:
```
import poster from "assets/images/home/lead/main-marketing-video-poster.jpg";
const placeholder: ExportedImageProps = {
  src: poster,
  alt: "GuestBell",
  sizes: posterSizes,
  priority: true,
  placeholder: "empty",
};
```
~~This is then passed to another component which is the one that renders the actual ExportedImage component. Might this be the reason? Naturally, it should still work, but I am thinking the fact that I load the image in a different file than where I use it could be messing up something in the build process.~~

**Comment by PeterKottas** [comment]
A couple of new findings:
1. This behavior is non-deterministic so it might be difficult to replicate
2. It's apparently caused (or rather announced) by a few things
a) You will see this error popup:
Image with src "/_next/static/media/main-marketing-video-poster.157bf3ec.jpg" has a "loader" property that does not implement width. Please implement it or use the "unoptimized" property instead.
b) If you pop into the networks tab, you will notice that the loading of the image was canceled:
![canceled](https://user-images.githubusercontent.com/10601911/219628302-59deee47-6f83-4d0e-bb78-1e651034ab25.jpg)
3. As a result, I assume the loading state of the image changes to error [here](https://github.com/Niels-IO/next-image-export-optimizer/blob/master/src/ExportedImage.tsx#L175)
4. That will cause the loader to switch to the fallback loader [here](https://github.com/Niels-IO/next-image-export-optimizer/blob/master/src/ExportedImage.tsx#L213)
5. As a result, only the original version is loaded 

So the culprit as it seems is the canceled loading of the image, why this happens I assume is the question here.
Also, we could possibly be more lenient when determining the onError? Maybe let canceled through? Although still, not sure what happens then. Will it actually load the correct image after the canceled one? Does this maybe have something to do with layout movement? Like ... it first wants to load some resolution, then it changes its mind mid-way, and decides to load a different one. Cancels the prior ... maybe that's what's going on? Maybe that's what's making it flaky.

**Comment by Niels-IO** [comment]
Removing the error handling as it is currently would be very dangerous. Right now, it always falls back to the original src, which is good behavior.

Have you noticed the bug also in production or only in the dev mode?

**Comment by PeterKottas** [comment]
Yeah, removing it certainly isn't an option. Just did that for us for the time being as we can pretty much guarantee the ok images are there. Struggling to find a way to tell fake errors apart from real ones. Yes production too.

**Comment by Niels-IO** [comment]
I can't reproduce the error right now, so I'm going to close this for the time being. If you figure out how to reproduce it, please give me a heads-up!

**Comment by PeterKottas** [comment]
Yeah, I still see this issue with the current version. I am using my fork where the error handling is disabled. No idea what this is about. Maybe somebody in the future will find this and be the wiser. The problem is people might be getting this, but never realizing it as it's not reliable. I only got it ~10% time, and only noticed when Lighthouse reported bad results for image compression. Subsequently, I've also replicated directly by testing on my machine. That's all I can give you so far. Feel free to keep it closed, I'll reopen if I find something. What I would say though, it might not be a terrible idea to add a flag that disables this fallback.
